tokens/src/core/components/general/link.json

```diff
@@ -1,27 +1,28 @@
 {
+  "$type": "textDecoration",
```

Comment:
[clarification] Similar to other comment about the `dimension` type defined more globally in a file, will _all_ tokens in this file be related to text-decoration? It would seem perhaps the `$type` should be defined within the `decoration` sub-object?

Reply:
Resolved in https://github.com/openedx/paragon/pull/3207